### PR TITLE
Return issue subscription status from API subscribe

### LINKED ISSUE #10958: [Feature] [API] IssueSubscription inidicate if already subscribed
instead of 201 return 200 if already subscribed

### PR DESCRIPTION
**Pull request by 6543**
close #10958
## ToDo
 * [x] indicate
 * [x] fix tests
 * [x] extend test

### CODE REVIEW

**Comment by 6543** [comment]
depend on #10967 to add tests

**Comment by 6543** [comment]
@guillep2k can you add this to v1.12 too - pleace?

**Comment by 6543** [comment]
still blocked by #10967

**Comment by 6543** [comment]
@techknowlogick you can remove wip now :)

File: routers/api/v1/repo/issue_subscription.go
Comment (zeripath) [inline routers/api/v1/repo/issue_subscription.go]:
```suggestion
	//     description: Successfully Subscribed
```

File: routers/api/v1/repo/issue_subscription.go
Comment (zeripath) [inline routers/api/v1/repo/issue_subscription.go]:
```suggestion
	//     description: Successfully Unsubscribed
```

File: templates/swagger/v1_json.tmpl
Comment (zeripath) [inline templates/swagger/v1_json.tmpl]:
```suggestion
            "description": "Successfully Subscribed"
```

File: templates/swagger/v1_json.tmpl
Comment (zeripath) [inline templates/swagger/v1_json.tmpl]:
```suggestion
            "description": "Successfully Unsubscribed"
```

**Review by zeripath — APPROVED** [review]
Minor nits

**Comment by 6543** [comment]
ready to merge 🚀

**Review by guillep2k — APPROVED** [review]

**Comment by guillep2k** [comment]
@6543 make sure you enable "allow edit by maintainers" so we can update the branch and merge.

image

**Comment by 6543** [comment]
@guillep2k it normaly is ... but I normaly have added you to the fork as colaborator too...

**Comment by guillep2k** [comment]
@6543 is this currently checked? (in the sidebar at the right) You can change it anytime.

image

**Comment by 6543** [comment]
@guillep2k IF I COULD I WOULD:
Screenshot_20200426-135143_1

**Comment by 6543** [comment]
can be merged :rocket:

**Comment by 6543** [comment]
ready to merge :rocket:

**Comment by 6543** [comment]
ready to merger :rocket:

**Comment by 6543** [comment]
ping lgtm

**Pull request merged**
